modules/configure_user/scripts/add_user.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -e
+
+# shellcheck disable=SC2154
+token="$(echo "$iam_token" | awk '{print $2}')"
+
+# decode the iam token
+jwt_decode(){
+    jq -R 'split(".") | .[1] | @base64d | fromjson' <<< "$1"
+}
+
+token_decoded="$(jwt_decode "$token")"
+email="$(echo "$token_decoded" | jq -r .email)"
+iam_id="$(echo "$token_decoded" | jq -r .iam_id)"
+display_name="$(echo "$token_decoded" | jq -r .name)"
+bss_account_id="$(echo "$token_decoded" | jq -r .account.bss)"
+
+# shellcheck disable=SC2154
+if [ "$location" == "us-south" ]; then
+    dataplatform_api="https://api.dataplatform.cloud.ibm.com"
+elif [ "$location" == "eu-gb" ]; then
+    dataplatform_api="https://api.eu-uk.dataplatform.cloud.ibm.com"
```

Comment:
This should be `api.eu-gb.dataplatform...` instead of `api.eu-uk..`

Reply:
good catch...thanks.